theano/tensor/nnet/tests/test_abstract_conv.py

```diff
@@ -139,19 +159,24 @@ def run_fwd(self, inputs_shape, filters_shape, ref=conv_corr,
 
         res_ref = numpy.array(f_ref())
         res = numpy.array(f())
+        log.debug("CorrMM Shape: %r", res_ref.shape)
+        log.debug("Abstr. conv2d Shape: %r", res.shape)
```

Comment:
Is this supposed to stay in the test? Won't `utt.assert_allclose` check the shapes anyway?